src/identify/index.js

```diff
@@ -45,16 +45,28 @@ class IdentifyService {
   /**
    * @constructor
    * @param {object} options
-   * @param {Registrar} options.registrar
+   * @param {PeerStore} options.peerStore
+   * @param {ConnectionManager} options.connectionManager
```

Comment:
This should probably just take `libp2p`, similar to the direction we're going with other services

Reply:
I am doing it in #612 . Can we wait for it instead?